bin/runtime/src/lib.rs

```diff
@@ -97,7 +97,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 71,
```

Comment:
Please use at least 73, as current Mainent has 72

Reply:
Or you can go straight ahead and use `13_300_000` here.

Reply:
done